medical_emr/__openerp__.py

```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
```

Comment:
Hi @yvaucher 

My understanding was that Odoo handles the major versioning, with the addons handling the minor versioning. Wouldn't we instead want `1.1`, which would present itself as `8.x.1.1` in Odoo?